misc/scripts/install-local.sh

```diff
@@ -361,6 +381,9 @@ case "$url" in
 		download "$url"
 		# hash the file
 		hashcheck "${url##*/}"
+		if [[ $? -ne 0 ]]; then
+			return 1
+		fi
```

Comment:
```suggestion	
		if ! hashcheck "${url##*/}"; then
			return 1
		fi
```

Reply:
yeah, this would be nice, because it looks nicer, and it gives minor speed improvements (even if it's very small